playbooks/acs.yml

```diff
@@ -51,6 +51,11 @@
   roles:
     - role: '../roles/search'
   post_tasks:
+    - name: Make sure Solr is running
+      service:
+        name: alfresco-search
+        status: started
```

Comment:
is this a new pattern we want to adhere with?
(ensure service are running only when running the playbook and not into the role tasks?)

Reply:
We discussed it some time ago and I though we agreed on that approach.
Keeping that check in the role would make the service being `started` by the task (can't be a handler if we want to always run it) and then `restart` by handlers in most cases (unless there is an unexpected issue preventing the handler from being triggered).
I think that's sub-optimal and could maybe be problematic with repo service when the service first initialize the system and place clusterLock on DB.

I initially didn't think of it as a general pattern we should adhere to but your comment makes me wonder to which extend service management should be removed from the roles themselves and added to the main play?

In any case I wouldn't do that now but maybe keep it for the extensibility work

Reply:
> We discussed it some time ago and I though we agreed on that approach.

repetita iuvant :D

> makes me wonder to which extend service management should be removed from the roles themselves and added to the main pla

that's probably something we need to ask to our users and how they expect after a playbook run